src/components/FoodBox.js

```diff
@@ -0,0 +1,54 @@
+// Library
+import React from 'react';
+
+class FoodBox extends React.Component {
```

Comment:
indentation on some files set as 2 and in some set as 4. Remember to set all to 2. Not that important here but double check when working on your projects.